evm-adapters/src/sputnik/cheatcodes/cheatcode_handler.rs

```diff
@@ -569,6 +573,34 @@ impl<'a, 'b, B: Backend, P: PrecompileSet> CheatcodeStackExecutor<'a, 'b, B, P>
                 // encode the data as Bytes
                 res = ethers::abi::encode(&[Token::Bytes(decoded.to_vec())]);
             }
+            HEVMCalls::GetCode(inner) => {
+                self.add_debug(CheatOp::GETCODE);
+
+                #[derive(Deserialize)]
+                struct ContractFile {
+                    bin: String,
+                }
+
+                let parts = inner.0.split(':').collect::<Vec<&str>>();
+                let contract_file = parts[0];
+                let contract_name = if parts.len() == 1 {
+                    parts[0].replace(".sol", "")
+                } else {
+                    parts[1].to_string()
+                };
+
+                let outdir = ProjectPathsConfig::find_artifacts_dir(Path::new("./"));
+                let path = outdir.join(format!("{}/{}.json", contract_file, contract_name));
```

Comment:
OK this is looking good. @mattsse WDYT?

Reply:
for now lgtm, but `Path::new("./")` is not necessarily the project root. 
 
can soon be replaced with using the config's value, but before that, we'd need to think about how to share that efficiently